### PR TITLE
Export PPPL data for demo purposes

### PR DESCRIPTION
**Pull request by bess**

### CODE REVIEW

File: app/services/s3_query_service.rb [hunk 1]
```diff
@@ -119,6 +119,7 @@ def find_s3_file(filename:)
   # Retrieve the S3 resources uploaded to the S3 Bucket
   # @return [Array<S3File>]
   def client_s3_files(reload: false, bucket_name: self.bucket_name)
+    return [] if Rails.env.development?
```
Comment (jrgriffiniii) [inline app/services/s3_query_service.rb:122]: 👍

File: spec/system/data_migration/dynamic_tension_form_submission_spec.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+      # Check that RORs were persisted as funder names
+      funders = dynamic_tension_work.resource.funders.map(&:funder_name).uniq
+      expect(funders).to contain_exactly("United States Department of Energy")
+
+      # # Ensure the datacite record produced validates against our local copy of the datacite schema.
+      # # This will allow us to evolve our local datacite standards and test our records against them.
+      datacite = PDCSerialization::Datacite.new_from_work(dynamic_tension_work)
+      expect(datacite.valid?).to eq true
+      expect(datacite.to_xml).to be_equivalent_to(File.read("spec/system/data_migration/dynamic_tension.xml"))
```
Comment (jrgriffiniii) [inline spec/system/data_migration/dynamic_tension_form_submission_spec.rb:85]: 👍

**Review by hectorcorrea — CHANGES_REQUESTED** [review]
See note regarding `return [] if Rails.env.development?`

File: app/services/s3_query_service.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (bess) [reply app/services/s3_query_service.rb]: @hectorcorrea and I came up with a better solution, which is now documented in the README file.

**Review by hectorcorrea — APPROVED** [review]

**Pull request merged**